posting/views.py

```diff
@@ -1,3 +1,46 @@
-from django.shortcuts import render
+import json
+import math
+
+from django.http           import JsonResponse, HttpResponse
+from django.views          import View
+from django.db.models      import Q
+from django.core.paginator import Paginator
+
+from posting.models   import (
+            Posting, JobCategory, Occupation,
+            Company, CompanyImage, CompanyDetail,
+            County, State, Tag, TagDetail,
+            CompanyTag, BookMark, Like
+                            )
+
+class DetailPaginationView(View):
+    def get(self, request, posting_id):
+        PER_PAGE = 4
+        try:
+            posting         = Posting.objects.get(id=posting_id)
+            posting_related = Posting.objects.filter(job_category=posting.job_category).exclude(id=posting_id)
+            limit_page      = math.ceil(posting_related.count()/PER_PAGE)
+            page            = request.GET['page']
+            paginator       = Paginator(posting_related, PER_PAGE)
+
+            if int(page) > limit_page:
+                return JsonResponse({"message": "NO_MORE_POSTING"}, status=404)
+            
+            posting_related = paginator.get_page(page)
+            posting_list    = [
+                        {
+                        'id': posting_related.id,
+                        'img': [image.image_url for image in posting_related.company_detail.company.companyimage_set.all()][0],
+                        'like': posting_related.like_set.all().count(),
```

Comment:
```suggestion
                        'like': posting_related.like_set.count(),
```
all()을 안 하고 바로 count()을 사용할 수 있습니다. 확인해 보세요.

Reply:
넵, 감사합니다!